hw/ip_templates/alert_handler/data/alert_handler.tpldesc.hjson

```diff
@@ -51,5 +51,11 @@
       type: "object"
       default: []
     }
+    {
```

Comment:
This should be unnecessary if we fix the VLNV convention :)  I guess the problem you face is that currently this name is not going through `instance_vlnv`?

Reply:
This was a conscious choice to not totally forbid running ipgen outside of topgen, actually.

At some point, this global package requirement should go away, though. It is not reasonable to require a globally defined `top_pkg::TL_<property>` to be consumed, since interconnection networks are not always uniform.